modules/bullet/space_bullet.cpp

```diff
@@ -582,36 +585,36 @@ void SpaceBullet::create_empty_world(bool p_create_soft_world) {
 	if (p_create_soft_world) {
 		world_mem = malloc(sizeof(btSoftRigidDynamicsWorld));
 	} else {
-		world_mem = malloc(sizeof(btDiscreteDynamicsWorld));
+		world_mem = malloc(sizeof(btDiscreteDynamicsWorldMt));
 	}
 
 	ERR_FAIL_COND_MSG(!world_mem, "Out of memory.");
 
 	if (p_create_soft_world) {
 		collisionConfiguration = bulletnew(GodotSoftCollisionConfiguration(static_cast<btDiscreteDynamicsWorld *>(world_mem)));
 	} else {
-		collisionConfiguration = bulletnew(GodotCollisionConfiguration(static_cast<btDiscreteDynamicsWorld *>(world_mem)));
+		collisionConfiguration = bulletnew(GodotCollisionConfiguration(static_cast<btDiscreteDynamicsWorldMt *>(world_mem)));
 	}
 
 	dispatcher = bulletnew(GodotCollisionDispatcher(collisionConfiguration));
 	broadphase = bulletnew(btDbvtBroadphase);
 	solver = bulletnew(btSequentialImpulseConstraintSolver);
+	int32_t process_count = OS::get_singleton()->get_processor_count() - 2;
+	solver_pool = bulletnew(btConstraintSolverPoolMt(process_count));
+	solver_mutithread = bulletnew(btSequentialImpulseConstraintSolverMt);
```

Comment:
classbtSequentialImpulseConstraintSolverMt is the single island solver. To gain performance may be helpful to tune this.

Reply:
I spent the last half hour reading through the forums on pybullet.org. I only found a handful of mentions of other solvers, and each and every time users were warned against using them since they were badly optimized.
As far as I can tell from what little information I found btSequentialImpulseConstraintSolverMt is the way to go.
Would appreciate it if someone who knows more about bullet could confirm this.